js/src/provider/WalletLinkProvider.ts

```diff
@@ -94,6 +95,10 @@ export class WalletLinkProvider
       ? options.walletLinkAnalytics
       : new WalletLinkAnalytics()
 
+    if (options.overrideIsCoinbaseWallet) {
```

Comment:
I think you need to check `if (typeof options.overrideIsCoinbaseWallet === 'boolean')`.

Otherwise `isCoinbaseWallet` will always be true since `overrideIsCoinbaseWallet = false` is falsey and won't set the `isCoinbaseWallet` variable to true.

Reply:
Great catch. I'll set the default in the constructor to avoid this.